prody/dynamics/plotting.py

```diff
@@ -211,7 +211,7 @@ def showProjection(ensemble, modes, *args, **kwargs):
       * 3 modes: :meth:`~mpl_toolkits.mplot3d.Axes3D.plot`"""
 
     import matplotlib.pyplot as plt
-
+    plt.figure()
```

Comment:
I was thinking of having it as a kwarg for each function. People can then decide each time they want to plot something whether they want a new plot or not. It shouldn't be an overall ProDy configuration thing.